### DISCUSSION TITLE
Messages in rule attributes are not working as expected.

### DISCUSSION THREAD

**Discussion by aftandilmmd** [livewire/livewire#6817] [7. Bugs (archived)]
### Livewire version

3.0.3

### Which PHP version are you using?

PHP 8.1

### Steps To Reproduce

1. Create **Test** component
2. Add any public value to the component. E.g **$title**
```
// app/Livewire/Test.php

#[Rule(rule: 'required', message: 'Required error')]
#[Rule(rule: 'max:5', message: 'Max error')]
#[Rule(rule: 'min:2', message: 'Min error')]
public string $title = '';
```

3. Add input and error to the component's balde view.
```
// resources/views/livewire/test.blade.php

<input wire:model.live="title" type="text" placeholder="Title" />
<div>
    @error('title') <span class="error">{{ $message }}</span> @enderror
</div>
```

4. Enter any text in the input field. It will always show last Rule's message (**"Min error"** message). 
```
input: "some long text"
Expected:  **'Max error'**
Output:  **'Min error'**
```
### Additional information

If we do not write a message value to the Rule attribute, Laravel's default messages work correctly.
```
// app/Livewire/Test.php

#[Rule('required')]
#[Rule('max:5')]
#[Rule('min:2')]
public string $title = '';
```

### Please confirm (incomplete submissions will not be addressed)

- [X] I have provided easy and step-by-step instructions to reproduce the bug.
- [X] I have provided code samples as text and NOT images.
- [X] I understand my bug report will be removed if I haven't met the criteria above.

**Comment by joshhanley** [comment]
@aftandilmmd ~~I think this has been fixed (maybe - I can't find the commit?). Going to tag a new release 3.0.4 shortly, can you upgrade once it has been released and test it out? Thanks!~~

Edit: This hasn't been fixed yet, but there is a failing test PR #6761
